docker-compose/redpanda-three-nodes.yml

```
@@ -108,18 +96,12 @@ services:
       - '172.24.1.4:33145'
       - --advertise-rpc-addr
       - '172.24.1.4:33145'
-      - --smp
-      - '1'
-      - --memory
-      - '1G'
-      - --reserve-memory
-      - '0M'
       - --overprovisioned
       - --check=false
-      - --default-log-level=debug
+      - --default-log-level=info
       - --seeds
       - 'redpanda1:33145'
-    image: 'docker.vectorized.io/vectorized/redpanda:v22.1.3'
+    image: 'docker.vectorized.io/vectorized/redpanda:v22.2.2'
```

```suggestion
    image: 'docker.vectorized.io/vectorized/redpanda:v22.2.4'
```

just released: https://github.com/redpanda-data/redpanda/releases/tag/v22.2.4